src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,13 @@
+@mixin grid-colums-count() {
```

Comment:
It seems like there is a typo in the mixin name `grid-colums-count`. According to the standard naming convention, it should be `grid-columns-count` with an 'n' in 'columns'.